server/app/controllers/applicant/EligibilityAlertSettingsCalculator.java

```diff
@@ -13,16 +13,19 @@
 import services.MessageKey;
 import services.program.ProgramNotFoundException;
 import services.program.ProgramService;
+import services.settings.SettingsManifest;
 
 public final class EligibilityAlertSettingsCalculator {
   private final ProgramService programService;
   private final MessagesApi messagesApi;
+  private final SettingsManifest settingsManifest;
 
   @Inject
   public EligibilityAlertSettingsCalculator(
-      ProgramService programService, MessagesApi messagesApi) {
+      ProgramService programService, MessagesApi messagesApi, SettingsManifest settingsManifest) {
```

Comment:
(blocking) Don't inject SettingsManifest here. Put the isNorthstar boolean on the calculate method instead. It will simplify the unit tests to a single matrix of parameters.